File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.cpp
int solution(int n) {
    int ans = 0;
    if(n % 2){
        for(int i = 1; i <= n; i += 2) ans += i;
    } else{
        for(int i = 2; i <= n; i += 2) ans += i * i;
    }
    return ans;
}

File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        if(n % 2 == 0){ // 짝
            for(int i = 2; i <= n; i += 2){
                answer += i * i;
            }
        } else{ // 홀
            for(int i = 1; i <= n; i += 2){
                answer += i;
            }
        }
        return answer;
    }
}